sources/tools/Stride.TextureConverter/Backend/Wrappers/FINetWrapper/Classes/MetadataModels.cs

```diff
@@ -1220,7 +1220,7 @@ public byte[] OECF
 
         /// <summary>
         /// Gets or sets the exposure bias. The unit is the APEX value.
-        /// Ordinarily it is given in the range of �99.99 to 99.99.
+        /// Ordinarily it is given in the range of 99.99 to 99.99.
```

Comment:
The encoding is showing a variation of '-' in Visual Studio, as it looks like it's meant to say "-99.99 to 99.99"